app/helpers/mustache_helper.rb

```diff
@@ -270,6 +270,67 @@ def navigation
     }
   end
 
+  def utility_nav
+    {
+      menu_id: "settings-menu",
+      style_modifier: "caret-right",
+      tabindex: "6",
+      items: [
+        {
+          url: "#",
+          text: "Settings",
+          icon: "settings",
+          submenu: {
+            items: [
+              {
+                text: "Settings",
+                subtitle: true,
+                url: false
+              },
+              {
+                text: "Language ",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.